MatomoTracker.php

```diff
@@ -1,30 +1,30 @@
 <?php
 /**
- * Piwik - free/libre analytics platform
+ * Matomo - free/libre analytics platform
  *
  * For more information, see README.md
  *
  * @license released under BSD License http://www.opensource.org/licenses/bsd-license.php
- * @link http://piwik.org/docs/tracking-api/
+ * @link https://matomo.org/docs/tracking-api/
  *
- * @category Piwik
- * @package PiwikTracker
+ * @category Matomo
+ * @package MatomoTracker
  */
 
 /**
- * PiwikTracker implements the Matomo Tracking Web API.
+ * MatomoTracker implements the Matomo Tracking Web API.
  *
  * For more information, see: https://github.com/matomo-org/matomo-php-tracker/
  *
- * @package PiwikTracker
+ * @package MatomoTracker
  * @api
  */
-class PiwikTracker
+class MatomoTracker
```

Comment:
Can you add a `class PiwikTracker extends MatomoTracker` or so for BC?

Reply:
Sure can do that. But actually I thought we will break BC nevertheless with Matomo 4

Reply:
I guess if we can keep BC so easily, there be no need for it maybe.

Reply:
done